setup.cfg

```diff
@@ -1,3 +1,78 @@
-# Inside of setup.cfg
 [metadata]
-description-file = README.md
+name = scib
+version = 0.2.0
+author = Malte Luecken, Maren Buettner, Daniel Strobl, Michaela Mueller
+author_email = malte.luecken@helmholtz-muenchen.de, michaela.mueller@helmholtz-muenchen.de
+description = Evaluate single cell integration methods
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+url = https://github.com/theislab/scib
+project_urls =
+    Pipeline = https://github.com/theislab/scib-pipeline
+    Reproducibility = https://theislab.github.io/scib-reproducibility
+Bug Tracker = https://github.com/theislab/scib/issues
+keywords = benchmark, single cell, data integration
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Topic :: Software Development :: Build Tools
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+package_dir =
+    = scib
+packages = find:
+include_package_data = true
+zip_safe = False
+python_requires = >=3.7
+install_requires =
+    numpy==1.18.1
+    pandas
+    seaborn
+    matplotlib
+    numba
+    scanpy>=1.5
+    anndata>=0.7.2
+    h5py<3
+    rpy2>=3
+    anndata2ri
+    scipy
+    scikit-learn
+    scikit-misc
+    loompy==3.0.6
```

Comment:
why do we need loompy? Is this a core dependency? Or is this related to conos or sth?

Reply:
Yes, I guess it was method specific so I'll remove it